### PR TITLE
Update CHI@NU site with new node info

### PR DESCRIPTION
**Pull request by diurnalist**

### CODE REVIEW

File: data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  "chassis": {
+    "manufacturer": "Dell Inc.",
+    "name": null,
+    "serial": null
+  },
+  "gpu": {
+    "gpu": false
+  },
+  "main_memory": {
+    "humanized_ram_size": "128 GiB",
```
Comment (diurnalist) [inline data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json:21]: This was hard-coded in the generator--is this right?

File: data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  },
+  "network_adapters": [
+    {
+      "bridged": false,
+      "device": "eno1",
+      "driver": null,
+      "interface": null,
+      "mac": null,
+      "management": null,
+      "model": "NetXtreme II BCM57800 1/10 Gigabit Ethernet",
```
Comment (diurnalist) [inline data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json:35]: I thought these had different NICs... maybe another generator fail.

File: data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    "cache_l2": 262144,
+    "cache_l3": 31457280,
+    "clock_speed": null,
+    "instruction_set": "x86-64",
+    "model": "Intel Xeon",
+    "other_description": "Intel(R) Xeon(R) CPU E5-2670 v3 @ 2.30GHz",
+    "vendor": "Intel",
+    "version": "E5-2670 v3"
+  },
+  "storage_devices": [
```
Comment (diurnalist) [inline data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json:96]: 😭

File: data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json (hunk 1 above)
Comment (codyhammock) [reply data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json:21]: Yeah, I'm not sure why that is. Seems pretty bad

**Pull request merged**